### PR TITLE
chore(release): prepare release notes for Modules 22.10

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [x] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -17,6 +17,24 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+
+Release date: `soon`
+
+#### Bug fixes
+
+- [Server] Fixed an issue causing images in views to not be displayed after an update.
+- [Server] Fixed an issue causing MAP server to crash at start up due to service group loading.
+- [Server] Fixed an issue preventing to save a map with "not a diagram" error message.
+- [Server] Fix critical error when editing a geo view.
+- [Server] Enable to parse metric from broker neb packets.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:30]:
```suggestion
- [Server] Enable to parse metric from Broker NEB packets.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -18,6 +18,24 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+
+Release date: `soon`
+
+#### Bug fixes
+
+- [Server] Fixed an issue causing images in views to not be displayed after an update.
+- [Server] Fixed an issue causing MAP server to crash at start up due to service group loading.
+- [Server] Fixed an issue preventing to save a map with "not a diagram" error message.
+- [Server] Fix critical error when editing a geo view.
+- [Server] Enable to parse metric from broker neb packets.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:31]:
```suggestion
- [Server] Enable to parse metric from Broker NEB packets.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -18,6 +18,24 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UI] Fixed view freezes beyond 500 elements.
+
+#### New Features
+
+- [UI] Change the label of the metric link modal.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:36]:
```suggestion
- [UI] You can now change the label of the metric link modal.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -18,6 +18,24 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [UX] Allow user to duplicate maps.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:37]:
```suggestion
- [UX] You can now duplicate maps.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -17,6 +17,24 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [UI] Fixed view freezes beyond 500 elements.
+
+#### New Features
+
+- [UI] Change the label of the metric link modal.
+- [UX] Allow user to duplicate maps.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:36]:
```suggestion
- [UX] You can now duplicate maps.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -17,6 +17,24 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Server] Fixed an issue causing images in views to not be displayed after an update.
+- [Server] Fixed an issue causing MAP server to crash at start up due to service group loading.
+- [Server] Fixed an issue preventing to save a map with "not a diagram" error message.
+- [Server] Fix critical error when editing a geo view.
+- [Server] Enable to parse metric from broker neb packets.
+- [UI] Fixed view freezes beyond 500 elements.
+
+#### New Features
+
+- [UI] Change the label of the metric link modal.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:35]:
```suggestion
- [UI] You can now change the label of the metric link modal.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -179,6 +206,15 @@ Release date: `October 26, 2022`
 
 ## Centreon MBI
 
+### 22.10.2
+
+Release date: `soon`
+
+#### Bug fixes
+
+- [Core] Improved database column size.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:215]: Enhancement?

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 8]
```diff
@@ -180,6 +207,15 @@ Release date: `October 26, 2022`
 
 ## Centreon MBI
 
+### 22.10.2
+
+Release date: `soon`
+
+#### Bug fixes
+
+- [Core] Improved database column size.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:216]: Enhancement?

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 9]
```diff
@@ -227,6 +263,14 @@ Release date: `October 26, 2022`
 
 ## Centreon License Manager
 
+### 22.10.1
+
+Release date: `soon`
+
+#### Enhancements
+
+- [UI] Improved upload of licenses modale.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:272]:
```suggestion
- [UI] Improved license upload modal.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md (hunk 7 above)
Comment (tuntoja) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:215]: only a technical one, that fixes potential issues related to this column in the database.

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by nmagniezCentreon — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**